redeal/redeal.py

```diff
@@ -221,6 +221,15 @@ def __call__(self, arg):
 controls.__name__ = "controls"
 
 
+def predeal_dict(*args):
```

Comment:
I think I would rather fix `__main__` and `redeal` so that the Deal class itself just takes a str->str mapping and does the conversion to Hands itself (so that you can just do `Deal.prepare({"S": "AK - - -"})` etc.).

Reply:
Yes, that would be cleaner.